examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,46 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+```html
```

Comment:
@wheresbarney thanks.

I do not see the header bidding auth error. This is what I see now:

```
{
   "request_id": "6261999187131519857",
   "errors": [
      "1798405500476623_1798416553808851 : "
   ],
   "bids": {
      
   }
}
```

My test app setup is pretty naive. From what you see is there any config or setup I am missing in order to receive test bids?

I do see this message in the AN UI console:

> Placement ID:	1798405500476623_1798416553808851
> Status:	Enabled for testing and received ad request recently